public/js/main.js

```diff
@@ -39,4 +39,6 @@ document.getElementById('prev-button').onclick = function () {
   addDateEventListeners();
 };
 
-getEventsModule({}, renderModule);
+getEventsModule({
+  date_from: new Date().toISOString().replace(/T.*/, '').split('-').join('-')
```

Comment:
- `.split('-').join('-')` effectively does nothing
- Instead of regex, it's easier to just `.slice(0, 10)`